### PR TITLE
Fix Build Break in Samples Repo

### PR DESCRIPTION
**Pull request by MSFTFox**
Added missing files to xcode project and re-ran VSImporter.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc-samples/36)

### CODE REVIEW

File: Scenarios/CGCT/CGCT/DemoScenarioText.m [hunk 1]
```diff
@@ -122,9 +122,9 @@ - (void)drawDemoIntoContext:(CGContextRef)context withFrame:(CGRect)bounds {
     NSRange fontRange = [textBox rangeOfString:@"font"];
     NSRange masterRange = [textBox rangeOfString:@"MASTER"];
 
-    CTFontRef mainFont = CTFontCreateWithName(@"Times New Roman", bounds.size.height * .04, NULL);
-    CTFontRef fontRangeFont = CTFontCreateWithName(@"Blackadder ITC", bounds.size.height * .04, NULL);
-    CTFontRef masterRangeFont = CTFontCreateWithName(@"Magneto Bold", bounds.size.height * .04, NULL);
+    CTFontRef mainFont = CTFontCreateWithName(CFSTR("Times New Roman"), bounds.size.height * .04, NULL);
```
Comment (msft-Jeyaram) [inline Scenarios/CGCT/CGCT/DemoScenarioText.m:125]: 👍

**Review by msft-Jeyaram — APPROVED** [review]

File: Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Debug-xcvars.txt [hunk 2]
```diff
@@ -90,7 +90,7 @@ PRODUCT_NAME = CGCT
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = CGCT
 PROJECT_DIR = ../../
-PROJECT_FILE_PATH = ../../CGCT.xcodeproj
```
Comment (DHowett-MSFT) [inline Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Debug-xcvars.txt:93]: relative paths.

**Review by DHowett-MSFT — CHANGES_REQUESTED** [review]

File: Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Debug-xcvars.txt (hunk 2 above)
Comment (MSFTFox) [reply Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Debug-xcvars.txt:93]: Woops, missed one. Good catch.

File: Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Release-xcvars.txt [hunk 3]
```diff
@@ -89,7 +89,7 @@ PRODUCT_NAME = CGCT
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = CGCT
 PROJECT_DIR = ../../
-PROJECT_FILE_PATH = ../../CGCT.xcodeproj
+PROJECT_FILE_PATH = D:\island\github\henryfox\WinObjC-Samples\Scenarios\CGCT\CGCT.xcodeproj
```
Comment (DHowett-MSFT) [inline Scenarios/CGCT/CGCT.vsimporter/CGCT-WinStore10/CGCT-Release-xcvars.txt]: 💠???

**Review by DHowett-MSFT — CHANGES_REQUESTED** [review]

**Review by DHowett-MSFT — APPROVED** [review]

**Pull request merged**